client/packages/coldchain/src/Equipment/api/hooks/document/useAssetUpdate.ts

```diff
@@ -1,13 +1,18 @@
 import { useMutation, useQueryClient } from '@openmsupply-client/common';
 import { useAssetApi } from '../utils/useAssetApi';
 import { DraftAsset } from '../../../types';
+import { useLocationApi } from '@openmsupply-client/system/src/Location/api/hooks/utils/useLocationApi';
 
 export const useAssetUpdate = () => {
   const queryClient = useQueryClient();
   const api = useAssetApi();
+  const locationApi = useLocationApi();
 
   return useMutation(async (asset: Partial<DraftAsset>) => api.update(asset), {
-    onSuccess: id => queryClient.invalidateQueries(api.keys.detail(id)),
+    onSuccess: id => {
+      queryClient.invalidateQueries(locationApi.keys.list());
```

Comment:
This is a bit of an overkill to remove all lists, but don't want the additional complexity of trying to match two types of query keys here?

Reply:
Yeah I think this is fine 😃